### DISCUSSION TITLE
Welcome to MAIZSIM Discussions!

### DISCUSSION THREAD

**Discussion by DennisTimlin-USDA** [USDA-ARS-ACSL/MAIZSIM#34] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bronevet-abc** [comment]
Hi, your README mentions that there's a way to compile the code under Linux but I don't see a Makefile. Is this still supported and if so, can you please point me to the appropriate instructions? Thanks!

**Comment by DennisTimlin-USDA** [comment]
Dear Bronevet,

Until recently I never compiled it myself on Linux, I left that to those that had experience. I just made sure there were not any functions not supported by recent linux and fortran compilers. I also had to make sure case was consistent for the names of the inserted files. Therefore, I didn't include a make file but assumed anyone interested would know how to do it.

Recently I  started working Linux in Docker containers and forced myself to learn how to compile in linux.
https://github.com/precision-sustainable-ag/BuildMaizsim

this repository has all the information you should need. Note that I have two versions of the model that I plan to merge in the future. one version works with our interface CLASSIM and the other has a residue decomposition model and CO2 transport. This version is not on the maizsim github site yet but the source code is in the container.

But, the make file in the container will work with the maizsim on this github site.

I have two types of interfaces. One is a python based GUI called CLASSIM, the other is a series of VBA scripts that query an excel spreadsheet with sheets that act as tables. The VBA code will build all the files needed to run the model except for weather (I am working on that now).

please feel free to contact me with more questions.

Best wishes,
Dennis


Feel free to contact me for more information.
________________________________
